website/documentation/guides/administer-shoots/scalability/_index.md

```diff
@@ -0,0 +1,141 @@
+
+**Cloud provider capacity**:
+
+Most of the time, `Nodes` in Kubernetes translate to virtual machines on a hyperscaler. An attempt to add more `Nodes` to a cluster might fail due to capacity issues resulting in an error message like this:
+
+```
+Cloud provider message - machine codes error: code = [Internal] message = [InsufficientInstanceCapacity: We currently do not have sufficient <instance type> capacity in the Availability Zone you requested. Our system will be working on provisioning additional capacity. 
+```
+
+In heavily utilized regions individual clusters are competing for scarce resources. So before choosing a region / zone, try to ensure the hyperscaler supports your anticipated growth. This might be done through quota requests or by contacting the respective support teams.
+To mitigate such a situation you may configure a worker pool with a different `Node` type and a corresponding [priority expander](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/expander/priority/readme.md) as part of a [shoot's autoscaler section](https://gardener.cloud/docs/gardener/api-reference/core/#core.gardener.cloud/v1beta1.ClusterAutoscaler). Please consult the [documentation](https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/FAQ.md#what-are-expanders) for more details.
+
+**Rolling of `Node` pools**:
+
+The overall number of `Nodes` is affecting the duration of a  cluster's maintenance. When upgrading a `Node` pool to a new OS image or Kubernetes version, all machines will be drained and deleted and replaced with new ones. The more `Nodes` a cluster has, the longer this process will take given that workloads are typically protected by `PodDisruptionBudgets`. Check the [documentation](https://gardener.cloud/docs/gardener/usage/shoot_updates/#customize-rolling-update-behaviour-of-shoot-worker-nodes) for details. Be sure to take this into consideration when planning maintenance.
+
+**Root disk**:
+
+You should be aware that the `Node` configuration impacts your workload's performance too. Take the root disk of a `Node` for example. While most hyperscalers offer the usage of HDD and SSD disks, it is strongly recommended to use SSD volumes as root disks. When there are lots of `Pods` on a `Node` or workloads making extensive use of `emptyDir` volumes, disk throttling becomes an issue otherwise. When a disk hits its IOPS limits, processes are stuck in IO-wait and slow down significantly. This can lead to a slow-down in kubelet's heartbeat mechanism and result in `Nodes` being replaced automatically as they appear to be unhealthy. To analyze such a situation you might have to run tools like `iostat`, `sar` or `top` directly on a `Node`. 
```

Comment:
```suggestion
You should be aware that the `Node` configuration impacts your workload's performance too. Take the root disk of a `Node`, for example. While most hyperscalers offer the usage of HDD and SSD disks, it is strongly recommended to use SSD volumes as root disks. When there are lots of `Pods` on a `Node` or workloads making extensive use of `emptyDir` volumes, disk throttling becomes an issue. When a disk hits its IOPS limits, processes are stuck in IO-wait and slow down significantly. This can lead to a slow-down in the kubelet's heartbeat mechanism and result in `Nodes` being replaced automatically, as they appear to be unhealthy. To analyze such a situation, you might have to run tools like `iostat`, `sar` or `top` directly on a `Node`. 
```